review/tests.py

```diff
@@ -2,38 +2,30 @@
 from .models import Review, DayCare, DogOwner
 from django.core.validators import ValidationError
 
-REVIEW = 'sample review'
-RATING = 5
-DAY_CARE_ID = 1
-DOG_OWNER_ID = 1
 
-
-@pytest.mark.django_db
-@pytest.fixture
-def test_review():
-    return Review.create(review=REVIEW, rating=RATING, daycare_id=DAY_CARE_ID, dogowner_id=DOG_OWNER_ID)
-
-
-@pytest.mark.django_db
+@pytest.mark.django_db()
 class TestReviewModel:
-    def test_persist_review(self, test_review):
-        assert test_review.review == REVIEW
+    def test_persist_review(self, test_review, test_review_data):
+        assert test_review.review == pytest.REVIEW
         assert test_review in Review.objects.all()
+        assert 1 == 1
 
     def test_del_review(self, test_review):
         test_review.delete()
         assert test_review not in Review.objects.all()
 
-    def test_rating_limit_min(self):
+    def test_rating_limit_min(self, test_review_data):
         with pytest.raises(ValidationError):
-            Review.create(review=REVIEW, rating=-1, daycare_id=DAY_CARE_ID, dogowner_id=DOG_OWNER_ID)
+            Review.create(review=pytest.REVIEW, rating=-1, daycare_id=pytest.DAY_CARE_ID,
+                          dogowner_id=pytest.DOG_OWNER_ID)
 
-    def test_rating_limit_max(self):
+    def test_rating_limit_max(self, test_review_data):
```

Comment:
use parametrize to have `test_rating_limit_min` and `test_rating_limit_max` tested on the same test.
also: is it relevant for this PR?